checks/evaluation/fuzzing_test.go

```diff
@@ -207,7 +208,8 @@ func TestFuzzing(t *testing.T) {
 		tt := tt
 		t.Run(tt.name, func(t *testing.T) {
 			t.Parallel()
-			if got := Fuzzing(tt.args.name, tt.args.findings); !cmp.Equal(got, tt.want, cmpopts.IgnoreFields(checker.CheckResult{}, "Error")) { //nolint:lll
+			dl := scut.TestDetailLogger{}
```

Comment:
should this also have TODO?

Didn't these functions previously take a DL, so we can grab the code from history?

Reply:
will look into it. Good idea. Lmk about https://github.com/ossf/scorecard/pull/3409#discussion_r1300730164